src/scanForInputs.js

```diff
@@ -37,17 +38,18 @@ const scanForInputs = (DeviceInterface) => {
         } else {
             // if this form is an ancestor of an existing form, remove that before adding this
             const childForm = [...forms.keys()].find((form) => parentForm.contains(form))
+            forms.get(childForm)?.destroy()
```

Comment:
Here we were not removing the listeners, so we ended up attaching the tooltip to a form object that was no longer in our `Map`. This caused weird bugs where we had a tooltip but couldn't click on it. I've introduced the `destroy` method that removes listeners and whatnot.